addons/web/static/src/views/relational_model.js

```diff
@@ -329,7 +329,7 @@ class DataPoint {
     _parseServerValue(field, value) {
         switch (field.type) {
             case "char": {
-                return value || "";
+                return value;
```

Comment:
remove the case then :D I was pretty sure we had to do that (and to default numeric values to 0). I must be mistaken